app/services/share_progress_variant_builder.rb

```diff
@@ -1,17 +1,17 @@
 require 'share_progress'
 
 class ShareProgressVariantBuilder
-  def self.create(params, variant_type:, page:, url:)
-    new(params, variant_type, page, url).create
+  def self.create(params:, variant_type:, page:, url:)
+    new(params, variant_type, page, url, nil).create
```

Comment:
What are these nils that get passed in this method and in the one below this one?